lib/unleash/configuration.rb

```diff
@@ -40,9 +40,9 @@ def metrics_interval_in_millis
     def validate!
       return if self.disable_client
 
-      raise ArgumentError, "URL and app_name are required parameters." if self.app_name.nil? || self.url.nil?
+      raise ArgumentError, "app_name is a required parameter." if self.app_name.nil?
```

Comment:
Bootstrap only should be a viable option (we can do this in the others), relaxing this constraint makes testing a ton easier but I may be able to roll this one back if we need

Reply:
Valid point.